### DISCUSSION TITLE
how can I edit the desktop right click create new menu?

### DISCUSSION THREAD

**Discussion by happmaoo** [lxqt/lxqt#2503] [Q&A]
RT

**Comment by isf63** [comment]
Run `xdg-user-dir TEMPLATES` in a terminal (or by going to LXQt Session Settings -> User Directories) to determine where your Templates directory is.
Then, add files to the Templates directory and they will appear as options under Create New -> ___. The file contents of these templates will also be copied into the newly created file.
![screenshot](https://github.com/lxqt/lxqt/assets/121320947/bbefc480-ecd1-4732-9f9d-79d0b692ddfd)
